src/common/constants.rs

```diff
@@ -11,3 +11,4 @@ pub const LINK_HOVER_STATE: &str = "hover:text-granola-orange hover:underline ho
 pub const TABLE_COL_NUMERIC_WIDTH: &str = "100px";
 pub const TABLE_COL_SHORT_WIDTH: &str = "150px";
 pub const TABLE_COL_HASH_WIDTH: &str = "auto";
+pub const TABLE_COL_USERNAME_WIDTH: &str = "200rem";
```

Comment:
@n1tranquilla This should be "26em" because there are up to 26 possible characters in the username.